serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -0,0 +1,154 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-jwt"]
+= Using JSON Web Token authentication with {ProductShortName} and {ServerlessProductName}
+:context: serverless-ossm-jwt
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+// what benefits does this integration provide?
+
+.Prerequisites
+. Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+. Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+. Configure xref:../../serverless/networking/serverless-ossm.adoc#serverless-ossm[Using {ProductShortName} with {ServerlessProductName}].
+
+.Procedure
+. Create a Knative service with sidecar injection, or enable sidecar injection for an existing service.
++
+[IMPORTANT]
+====
+Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+====
++
+.. Create a Knative service with sidecar injection.
+... Copy the following into a YAML file:
++
+
+[source,yaml]
+----
+  apiVersion: serving.knative.dev/v1
+  kind: Service
+  metadata:
+    name: hello-example
+  spec:
+    template:
+      metadata:
+        name: hello-example-1
+        annotations:
+          sidecar.istio.io/inject: "true"
+      spec:
+        containers:
+        - image: gcr.io/knative-samples/helloworld-go
```

Comment:
Can you use `docker.io/openshift/hello-openshift` for the image?
`docker.io` is not official image, but it is better than gcr.io.